test/unit/carousel.tests.js

```diff
@@ -1,3 +1,4 @@
+/* eslint max-statements: 0 */
```

Comment:
is there a particular reason for this disable? not really blocking but curious if the tests could be broken up better?

Reply:
Eh, its mostly that I just find it to not a super useful rule for tests. since it will trigger just by having "too many tests". Sure you could fix it by breaking up into multiple files or mocha's describes. But really I find that anything to make it easier to write tests and get out of the way ends up with better / more test coverage. I'd rather drop anything that hinders testing at all.

Reply:
Fair enough for me :)